docs/user-guide/01-blueprint-reference.md

```diff
@@ -1242,6 +1243,46 @@ unselected = [ "grub2_password" ]
 </TabItem>
 </Tabs>
 
+### OpenSCAP JSON Tailoring
```

Comment:
please add this bullet for the overview (and then also the manual anchor is needed)
```suggestion
### OpenSCAP JSON Tailoring 🔵 {#openscap-json-tailoring}
```